molgenis-metadata-manager/src/main/frontend/src/store/actions.js

```diff
@@ -32,7 +32,12 @@ export default {
     get({apiUrl: '/api'}, '/v2/sys_md_EntityType?num=10000')
       .then(response => {
         const nonSystemEntities = response.items.filter(function (item) {
-          return !item.package.id.startsWith('sys')
+          // TODO we should filter on parent package is sys, a user can also select a package name starting with sys_
```

Comment:
And I did not understand the TODO :p

You don't want people to edit anything that us under the system metadata (won't work anyways)
So we block everything that is system, also those entities that are nested under system